servers/src/main/java/tachyon/web/WebInterfaceWorkerGeneralServlet.java

```diff
@@ -51,29 +85,45 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
    * @param request The HttpServletRequest object
    * @throws IOException
    */
-  // TODO: Provide a way to populate these values
   private void populateValues(HttpServletRequest request) throws IOException {
     request.setAttribute("debug", Constants.DEBUG);
 
-    // request.setAttribute("workerAddress", mWorkerStorage.getWorkerAddress().toString());
-    //
-    // request.setAttribute("uptime",
-    // Utils.convertMsToClockTime(System.currentTimeMillis() - mWorkerStorage.getStartTimeMs()));
-    //
-    // request.setAttribute("startTime", Utils.convertMsToDate(mWorkerStorage.getStartTimeMs()));
-    //
-    // request.setAttribute("version", Version.VERSION);
-    //
-    // request.setAttribute("capacityBytes",
-    // CommonUtils.getSizeFromBytes(mWorkerStorage.getCapacityBytes()));
-    //
-    // request.setAttribute("usedBytes",
-    // CommonUtils.getSizeFromBytes(mWorkerStorage.getUsedBytes()));
-    //
-    // request.setAttribute("capacityBytesOnTiers", mWorkerStorage.getCapacityBytesOnTiers());
-    //
-    // request.setAttribute("usedBytesOnTiers", mWorkerStorage.getUsedBytesOnTiers());
-    //
-    // request.setAttribute("storageDirs", mWorkerStorage.getStorageDirs());
+    request.setAttribute("workerAddress", mWorker.getWorkerAddress().toString());
+
+    request.setAttribute("uptime",
+        Utils.convertMsToClockTime(System.currentTimeMillis() - mWorker.getStartTimeMs()));
+
+    request.setAttribute("startTime", Utils.convertMsToDate(mWorker.getStartTimeMs()));
+
+    request.setAttribute("version", Version.VERSION);
+
+    BlockStoreMeta storeMeta = mWorker.getStoreMeta();
+    long capacityBytes = 0L;
+    long usedBytes = 0L;
+    List<Long> capacityBytesOnTiers = storeMeta.getCapacityBytesOnTiers();
+    List<Long> usedBytesOnTiers = storeMeta.getUsedBytesOnTiers();
+    for (int i = 0; i < capacityBytesOnTiers.size(); i ++) {
+      capacityBytes += capacityBytesOnTiers.get(i);
+      usedBytes += usedBytesOnTiers.get(i);
+    }
+
+    request.setAttribute("capacityBytes", CommonUtils.getSizeFromBytes(capacityBytes));
+
+    request.setAttribute("usedBytes", CommonUtils.getSizeFromBytes(usedBytes));
+
+    request.setAttribute("capacityBytesOnTiers", capacityBytesOnTiers);
+
+    request.setAttribute("usedBytesOnTiers", usedBytesOnTiers);
+
+    List<UiStorageDir> storageDirs = new ArrayList<UiStorageDir>();
```

Comment:
Since we know the total size, could we instantiate the array list with the size?